### ISSUE TITLE
Enable more tests

### ISSUE THREAD

**Issue by fabaff** [azogue/aiopvpc#11]
Without network access almost all tests are skipped.

```bash
[...]
-- Docs: https://docs.pytest.org/en/stable/warnings.html
================== 1 passed, 14 skipped, 11 warnings in 0.47s ==================
```

Would it be possible to run the tests with fixtures?

Most build systems for distribution packages don't allow network access during the build process of the packages. Thus, it would be nice to have some tests which could be run.

Thanks.

**Comment by azogue** [comment]
It seems related to the pytest environment when running the test suite:
```
=============================== warnings summary ===============================
tests/test_pvpc.py: 11 warnings
  /nix/store/69jp5jhv1v79zk2dlv5vy9ssz89bik9w-python3.8-pytest-6.2.2/lib/python3.8/site-packages/_pytest/python.py:172: PytestUnhandledCoroutineWarning: async def functions are not natively supported and have been skipped.
  You need to install a suitable plugin for your async framework, for example:
    - anyio
    - pytest-asyncio
    - pytest-tornasync
    - pytest-trio
    - pytest-twisted
    warnings.warn(PytestUnhandledCoroutineWarning(msg.format(nodeid)))

-- Docs: https://docs.pytest.org/en/stable/warnings.html
================== 1 passed, 14 skipped, 11 warnings in 0.47s ==================
```

* **"install a suitable plugin for your async framework"**

In other projects I've used `pytest-asyncio` in dev dependencies, but this one is using `pytest-aiohttp`, could that be the problem? 

Running tests locally (macOS) with a fresh poetry env is going ok, with 12 tests passing and 3 skipped (as these are the ones that make real calls)

**Comment by azogue** [comment]
closed by #13
